spec/encrypted_cookie_spec.rb

```diff
@@ -114,7 +115,7 @@ def app
     sleep 0.08
```

Comment:
I think this line should then be made '::TIME_TO_LIVE*0.8' (i.e. < than the time to live)